template.tpl

```diff
@@ -1357,11 +1364,15 @@ const generateConfiguration = () => {
 
     if (!!data.detAttribution) {
       initOptions.defaultTracking.attribution = {};
-
+      
```

Comment:
Is this intentional?

Reply:
I added some logs and deleted them after debugging. These indents are added by VSCode. Not sure why they were not there before.